Simplified/Book/UI/NYPLBookCellDelegate+Audiobooks.m

```diff
@@ -194,40 +189,6 @@ - (void)registerCallbackForLogHandler
   }];
 }
 
-// Non-thread safe: currently this is always called on the main thread.
-// Even more stricly, since NTPLBookCellDelegate is a singleton (!?), this
-// method should be called only when the previous audiobookViewController is
-// no longer used.
-- (void)scheduleTimerForAudiobook:(NYPLBook *)book
```

Comment:
this code was essentially translated to swift, correct?

Reply:
The main functionality (updating position in book registry) has been translated a Swift function in `NYPLLastListenPositionSynchronizer`, which is named `updateLastListenPositionInMemory()`. However, the timer is removed because we already have a timer in the `AudiobookManager` class with the same interval. So we now call the function through a protocol in an existing timer instead of creating a timer.